call.go

```diff
@@ -91,7 +91,18 @@ func (c *CalledChecker) From(b *ssa.BasicBlock, i int, receiver types.Type, meth
 	}
 
 	if !identical(v.Type(), receiver) {
-		return false, false
+		t, ok := v.Type().(*types.Tuple)
```

Comment:
@tetsuzawa 
Could you this code extract to a function?

Reply:
Do you mean extract the for statement to a function like this?

```go
/* ... */


	if !identical(v.Type(), receiver) {
		t, ok := v.Type().(*types.Tuple)
		if !ok {
			return false, false
		}
		if !identicalToTupleChild(t, receiver) {
			return false, false
		}
	}

/* ... */

func identicalToTupleChild(t *types.Tuple, receiver types.Type) bool {
	for j := 0; j < t.Len(); j++ {
		if identical(t.At(j).Type(), receiver) {
			return true
		}
	}
	return false
}
```

or should I move the type assertion to a function too?

Reply:
>should I move the type assertion to a function too?
yes

Reply:
I extract the code to function in 0fa96b4